pkg/pillar/types/attesttypes.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+//AttestNonce carries nonce published by requester
+type AttestNonce struct {
+	nonce     []byte
+	requester string
+}
+
+//Key returns nonce content, which is the key as well
+func (nonce AttestNonce) Key() []byte {
```

Comment:
Note that the key should be string so it can easily become the name (including filename) of the published object

Reply:
Changed it to string